src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -102,6 +88,8 @@ var ALL_SCOPES = [
  *   Custom function to obtain the credentials.
  * @param {grpc.ClientCredentials} opts.sslCreds
  *   A ClientCredentials for use with an SSL-enabled channel.
+ * @param {Object} opts.grpc
```

Comment:
No.

The consistency is the key -- the exactly same grpc module should be used among the API. Previous design was broken because this generated file loads grpc, and gax also loads grpc as well -- they happen to be same in a normal installation, but can be broken.

Now,
- if opts.grpc is not specified, GAX loads its own grpc module and that is used everywhere
- if opts.grpc is specified, GAX stops loading its own grpc and using specified one everywhere

So either way it should work (to make this work, we need to modify GAX as well, see https://github.com/googleapis/gax-nodejs/pull/30).